Tests/PubNubTests/Integration/SubscriptionIntegrationTests.swift

```diff
@@ -46,7 +46,7 @@ class SubscriptionIntegrationTests: XCTestCase {
     listener.didReceiveSubscription = { event in
       switch event {
       case let .connectionStatusChanged(status):
-        print("Status: \(status)")
+//        print("Status: \(status)")
```

Comment:
Is this left in intentionally?

Reply:
Yeah it's just for easy debugging of integration tests; not real code